https://git.netfilter.org/nftables/commit/?id=2e3c68f26d5bd60c8ea7467fa9018c282a7d8c47

From 2e3c68f26d5bd60c8ea7467fa9018c282a7d8c47 Mon Sep 17 00:00:00 2001
From: Jan Palus <jpalus@fastmail.com>
Date: Sat, 6 Dec 2025 00:43:58 +0100
Subject: build: fix ./configure with non-bash shell

 CONFIG_SHELL=/bin/dash ./configure

breaks with:

 ./config.status: 2044: Syntax error: Bad for loop variable

Fixes: 64c07e38f049 ("table: Embed creating nft version into userdata")
Signed-off-by: Jan Palus <jpalus@fastmail.com>
Signed-off-by: Pablo Neira Ayuso <pablo@netfilter.org>
---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index 6825474b..dd172e88 100644
--- a/configure.ac
+++ b/configure.ac
@@ -157,7 +157,7 @@ AC_CONFIG_COMMANDS([nftversion.h], [
 	echo "	${STABLE_RELEASE}"
 	echo "};"
 	echo "static char nftbuildstamp[[]] = {"
-	for ((i = 56; i >= 0; i-= 8)); do
+	for i in `seq 56 -8 0`; do
 		echo "	((uint64_t)MAKE_STAMP >> $i) & 0xff,"
 	done
 	echo "};"
-- 
cgit v1.2.3
